examples/language-modeling/README.md

```diff
@@ -478,7 +478,6 @@ LOWER_LIST=ops_bf16.txt python ../gaudi_spawn.py \
 	--world_size 8 --use_mpi run_lora_clm.py \
 	--model_name_or_path meta-llama/Llama-2-7b-hf \
 	--dataset_name tatsu-lab/alpaca \
-	--bf16 True \
```

Comment:
@MohitIntel why do you remove this arg here?

Reply:
Mistakenly removed. Will fix.
After discussion with Libin both bf16 and fp8 flags are indeed needed in the command.